docs/snippets/MULTI1.cpp

```diff
@@ -5,18 +5,25 @@ int main() {
 ov::Core core;
 std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
 ov::CompiledModel compileModel = core.compile_model(model, "MULTI:HDDL,GPU");
+
+// Once the priority list is set, you can alter it on the fly:
 // reverse the order of priorities
 compileModel.set_property(ov::device::priorities("GPU,HDDL"));
-// you can even exclude some device (HDDL here)
+
+// exclude some devices (in this case, HDDL)
 compileModel.set_property(ov::device::priorities("GPU"));
-// and then return it back
+
+// return excluded devices
 compileModel.set_property(ov::device::priorities("GPU,HDDL"));
```

Comment:
suggest change to :
// include back the excluded devices

Reply:
done with minor rewording :)